data/mods/Magiclysm/monster_factions.json

```diff
@@ -8,6 +8,11 @@
     "name": "dragon_black",
     "friendly": [ "ooze", "lizardfolk" ]
   },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "hunllef",
+    "base_faction": "herbivore"
```

Comment:
You've said they should be OK with herbivores (and I'm guessing with small animals?) but not like other things. How should other things react to them? As herbivores (which is the intent of the above) or what?

Reply:
I want them to be ornery which maybe that's a monster reaction we don't really have.   I wish we had something to be similar to how bulls generally tend to work where they charge you once and then go stop caring about you until you "challenge" them again.